keps/sig-node/4639-oci-volume-source/README.md

```diff
@@ -1088,20 +1140,31 @@ question.
 Pick one more of these and delete the rest.
 -->
 
-- [ ] Metrics
-  - Metric name:
-  - [Optional] Aggregation method:
-  - Components exposing the metric:
+- [x] Metrics
+  - Metric name: `kubelet_image_pull_duration_seconds_bucket`
+  - Components exposing the metric: kubelet
+- [x] Metrics
+  - Metric name: `pod_start_sli_duration_seconds`
+  - Components exposing the metric: kubelet
 - [ ] Other (treat as last resort)
   - Details:
 
+Note: there is not currently a well defined SLI[1] for stateful pods, and it's likely this feature will drastically affect that if a big
+image needs to be pulled from a registry.
+
+1: https://github.com/kubernetes/community/blob/master/sig-scalability/slos/pod_startup_latency.md#footnote3
+
 ###### Are there any missing metrics that would be useful to have to improve observability of this feature?
 
 <!--
 Describe the metrics themselves and the reasons why they weren't added (e.g., cost,
 implementation difficulties, etc.).
 -->
 
+A metric to show volume type so admins could find when their image volumes were requested
+as well as a success/error metric for showing that volumes did or did not get mounted
+These will be added in beta.
```

Comment:
At beta, are these missing or not?

Reply:
They currently don't exist, but will before this moves to beta